packages/Button/src/button.scss

```
@@ -37,10 +37,12 @@
   --ids-btn-background-secondary-disabled: var(--grey-300);
 
   /* Premium */
-  --ids-btn-text-premium: var(--samoyed);
-  --ids-btn-background-premium: var(--dandelion-500);
-  --ids-btn-background-premium-hover: var(--dandelion-600);
-  --ids-btn-background-premium-disabled: var(--dandelion-200);
+  --ids-btn-text-premium: var(--grey-800);
+  --ids-btn-text-premium-disabled: var(--grey-600);
```

dans le Figma la couleur est a --grey-500
```suggestion
  --ids-btn-text-premium-disabled: var(--grey-500);
```

---

Corrigé dans commit ce5bc53